numpy/_core/tests/test_strings.py

```diff
@@ -617,6 +690,80 @@ def test_replace_unicode(self, dt):
         assert_array_equal(np.strings.replace(buf,  "<", "&lt;", MAX),
                            "...\u043c......&lt;")
 
+    @pytest.mark.parametrize("dt", ["U", "T"])
+    @pytest.mark.parametrize("in_", [
+        '\U00010401',
+        '\U00010427',
+        '\U00010429',
+        '\U0001044E',
+        '\U0001D7F6',
+        '\U00011066',
+        '\U000104A0',
+        '\U0001F107',
+    ])
+    def test_isalnum_unicode(self, in_, dt):
+        # TODO: Remove this
+        if dt == "T":
+            pytest.xfail(
+                "StringDType support to be added in a follow-up commit")
+        assert_array_equal(np.strings.isalnum(in_), True)
+
+    @pytest.mark.parametrize("dt", ["U", "T"])
+    @pytest.mark.parametrize("in_,out", [
+        ('\u1FFc', False),
+        ('\u2167', False),
+        ('\U00010401', False),
+        ('\U00010427', False),
+        ('\U0001F40D', False),
+        ('\U0001F46F', False),
+        ('\u2177', True),
+        ('\U00010429', True),
+        ('\U0001044E', True),
+    ])
+    def test_islower_unicode(self, in_, out, dt):
+        # TODO: Remove this
+        if dt == "T":
+            pytest.xfail(
+                "StringDType support to be added in a follow-up commit")
+        assert_array_equal(np.strings.islower(in_), out)
+
+    @pytest.mark.parametrize("dt", ["U", "T"])
```

Comment:
Oh right. Although that wouldn't work for the main `TestMethods` class, since that's a class parametrization, so `xfail(strict=True)` would fail instantly. I'll do it for the parametrized methods though.